.gitignore

```diff
@@ -113,6 +113,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+harmony-ntz
```

Comment:
Just curious - what is this file?

Reply:
The readme recommended creating a virtual environment called harmony-ntz with pyenv-virtualenv. If you use venv to do the same, it creates a directory with that name for the virtual environment.

Reply:
I'm guessing people have not been using the virtual environment naming suggested in the readme in favor of something like "venv". (I used harmony-ntz when I created my virtual environment because that seemed to be what was being suggested.)

Reply:
I've been using `pyenv virtualenv`, but for consistency with ourselves and other projects I've seen and easier setup of `$PATH`, it may be better to just have the environment called `env` (or `venv` or you pick one) in every project, so we can run the same `source env/bin/activate` everywhere and have the same gitignore everywhere.